README.md

```diff
@@ -62,70 +100,66 @@ using       ton = unlib::kilo<kilogram>;       // humans don't use megagram
 A quantity combines a [unit](#units) (_mass_), a [scale](#scales) (_kilo_), a specific [value type](#values) (_double_), and an optional [tag](#tags) (_reactive_): 
 
 ```c++
-using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>. 
+using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>;
+using     watt = div_quantity_t<joule<V>,second<V>>;
 ```
 
-_(Note: All of unlib's predefined quantity types, including _kilogram_, are templates that take the value type – here **`double`** – as a template parameter.)_ 
+_(Note: All of unlib's predefined quantity types, including _kilogram_, are templates that take the value type – here fixed to **`double`** – as a template parameter.)_ 
 
-A quantity can be created from a value type explicitly: 
-
-```c++
-kilogram kg{42};
-```
-
-It cannot be implicitly created from a value type, though: 
+A quantity can be created from a value type explicitly. It cannot be implicitly created from a value type, though: 
 
 ```c++
 void f(unlib::quantity<unlib::mass,unlib::kilo,double>);
 
-f(kg);           // fine
+kilogram kg{42}; // fine
+f(kg);           // fine, too
 f(42);           // won't compile
 f(kilogram{42}); // also fine
 
 using namespace unlib::literals; // needed for unit suffixes
-f(42_kg);        // fine, too
+f(42_kg);        // also works
```

Comment:
```suggestion
f(42.0_kg);        // also works
f(23_kg);           // does not work
```
the integer literal will create an integer-valued quantity, which is not accepted by `f`, as there is no implicit value type cast.

Reply:
I would not bring the value type into this, as that is dealt with only later. But that was a good catch, I am using a double literal now.